ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -488,3 +488,10 @@ def __iter__(self):
 
     def __mod__(self, value, /):
         return np_frontend.mod(self, value, out=self)
+    
+    def ptp(self, *, axis=None, out=None, keepdims=False):
+        ret = ivy.max(self.ivy_array) - ivy.min(self.ivy_array)
+        if axis==int :
```

Comment:
Hi! Have you tried running the test locally? I think this should be causing an error. This is not how you should be checking if a variable is an integer. You could use `isinstance` instead.